src/Propel/Generator/Command/TestPrepareCommand.php

```diff
@@ -95,25 +95,30 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $result = static::CODE_SUCCESS;
         foreach ($this->fixtures as $fixturesDir => $connections) {
-            $this->buildFixtures(sprintf('%s/%s', self::FIXTURES_DIR, $fixturesDir), $connections, $input, $output);
+            $run = $this->buildFixtures(sprintf('%s/%s', self::FIXTURES_DIR, $fixturesDir), $connections, $input, $output);
+            if ($run !== static::CODE_SUCCESS) {
+                $result = $run;
+            }
```

Comment:
I really like this "un-masking" of error codes.

Reply:
so far the return value was unused, yeah :)